app/models/BusinessType.scala

```diff
@@ -53,6 +56,15 @@ object BusinessType extends Enumerable.Implicits {
     Radios(field, items)
   }
 
+  def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
```

Comment:
I am not comfortable with the view layer code mixed with the models but I can see that this code is consistent with the rest of the projects. I am happy to leave it as it is for now but perhaps we can think about how to achieve better separation.

Reply:
sure , we can look at it sometime.